ambari-server/src/main/resources/Ambari-DDL-SQLAnywhere-CREATE.sql

```diff
@@ -1314,6 +1318,7 @@ INSERT INTO ambari_sequences(sequence_name, sequence_value) values ('hostcompone
 INSERT INTO ambari_sequences(sequence_name, sequence_value) values ('mpack_inst_svc_id_seq', 0);
 INSERT INTO ambari_sequences(sequence_name, sequence_value) values ('mpack_instance_id_seq', 0);
 INSERT INTO ambari_sequences(sequence_name, sequence_value) values ('hostgroup_component_id_seq', 0);
+INSERT INTO ambari_sequences(sequence_name, sequence_value) values ('cnfgrpclstrcnfigmpg_id_seq', 0);
```

Comment:
This new sequence `cnfgrpclstrcnfigmpg_id_seq` is missing from `ambari-server/src/main/resources/Ambari-DDL-SQLServer-CREATE.sql`.

Reply:
Ok